src/main/java/teammates/common/util/EmailType.java

```diff
@@ -23,6 +23,7 @@ public enum EmailType {
     NEW_INSTRUCTOR_ACCOUNT("TEAMMATES: Welcome to TEAMMATES! %s"),
     STUDENT_COURSE_JOIN("TEAMMATES: Invitation to join course [%s][Course ID: %s]"),
     STUDENT_COURSE_REJOIN_AFTER_GOOGLE_ID_RESET("TEAMMATES: Your account has been reset for course [%s][Course ID: %s]"),
+    NEW_ACCOUNT_REQUEST_ADMIN_ALERT("TEAMMATES: New Account Request Received"),
```

Comment:
```suggestion
    NEW_ACCOUNT_REQUEST_ADMIN_ALERT("TEAMMATES (Action Needed): New Account Request Received"),
```

Reply:
Will this be more telling for admin to take actions

Reply:
I am supportive of this change. However, do we still need to check with the prof about this? If not, I can just make this change.

Reply:
Yep let's change it. Thanks!